src/main/groovy/com/google/gcloud/dsl/GCloudConfigurable.groovy

```diff
@@ -46,7 +47,7 @@ class GCloudConfigurable {
     }
 
     void args(Map<String, ?> args) {
```

Comment:
But maybe, I should have standardized against the wildcard. I'll take another look